lib/bundle-validator.js

```diff
@@ -18,7 +18,9 @@ var Eslint = require('eslint').CLIEngine;
 var Fs = require('fs');
 var sizeOf = require('image-size');
 var Path = require('path');
-var Validator = require('jsonschema').Validator;
+var Validator = require('ajv');
+const JsonSchemaValidatorOptions = {schemaId: 'auto', allErrors: true, verbose: true};
```

Comment:
🍹 Does it make any sense to make any of these configurable? My gut says no, but I figured it's worth asking.

Reply:
We can make it configurable based on feedback in the future.